IonDotnet/Tree/Impl/ValueFactory.cs

```diff
@@ -0,0 +1,152 @@
+﻿using System;
+using System.Numerics;
+
+namespace IonDotnet.Tree.Impl
+{
+    public class ValueFactory : IValueFactory
+    {
+        public ValueFactory()
+        {
+        }
+
+        public IIonBlob NewNullBlob()
+        {
+            return IonBlob.NewNull();
+        }
+
+        public IIonBlob NewBlob(ReadOnlySpan<byte> bytes)
+        {
+            return new IonBlob(bytes);
+        }
+
+        public IIonBool NewNullBool()
+        {
+            return IonBool.NewNull();
+        }
+
+        public IIonBool NewBool(bool value)
+        {
+            return new IonBool(value);
+        }
+
+        public IIonClob NullClob()
```

Comment:
`NullClob` -> `NewNullClob` for consistency.

Reply:
Done.